imports/ui/dialogs/create-report-dialog.jsx

```diff
@@ -25,12 +26,24 @@ class CreateReportDialog extends Component {
       }
     }, 300)
   }
+
+  placeholder () {
+    const { unitName } = this.props
+    const creationTime = moment().format('DD/MM/YYYY')
+    return `${creationTime} ${unitName}`
+  }
+
   handleSubmit = evt => {
     evt.preventDefault()
     const { reportTitle } = this.state
-    const { unitName } = this.props
-    this.props.dispatch(createReport(unitName, reportTitle))
+    const { unitName, dispatch } = this.props
+    if (reportTitle === '') {
+      dispatch(createReport(unitName, this.placeholder()))
```

Comment:
I chose to directly insert a placeholder, instead of updating the reportTitle state, because setState does not update immediately before dispatching. Dispatching with callback function in setState didn't work either.

This one works 😄😄